opencraft/settings.py

```diff
@@ -261,6 +261,7 @@
 OPENSTACK_SANDBOX_SSH_KEYNAME = env('OPENSTACK_SANDBOX_SSH_KEYNAME', default='opencraft')
 OPENSTACK_SANDBOX_SSH_USERNAME = env('OPENSTACK_SANDBOX_SSH_USERNAME', default='ubuntu')
 
+OPENSTACK_PRODUCTION_FLAVOR = env.json('OPENSTACK_PRODUCTION_FLAVOR', default={"ram": 8192, "disk": 80})
```

Comment:
@lgp171188 Also, let's call this `OPENSTACK_PRODUCTION_INSTANCE_FLAVOR` like @pomegranited suggested; that's a bit more specific than the current name :)